src/saml2/pack.py

```diff
@@ -39,37 +33,37 @@
         from elementtree import ElementTree
 import defusedxml.ElementTree
```

Comment:
This should not be altered as it overrides the default `xml.etree.ElementTree` functions, adding proper secure configuration for the xml parser. The whole code from line 22 to this one should be abstracted and handled differently and should happen soon based on #498